plugins/kubernetes-cluster/src/components/ApiResources/ApiResources.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { List, ListItem, ListItemText } from '@material-ui/core';
+import { useApiResources } from './useApiResources';
+import React from 'react';
+import { useEntity } from '@backstage/plugin-catalog-react';
+import { Skeleton } from '@material-ui/lab';
+import { InfoCard, Table, TableColumn } from '@backstage/core-components';
+import { IAPIGroup } from '@kubernetes-models/apimachinery/apis/meta/v1';
+
+const defaultColumns: TableColumn<IAPIGroup>[] = [
+  {
+    title: 'name',
+    highlight: true,
+    render: (apiGroup: IAPIGroup) => {
+      return apiGroup.name;
+    },
+  },
+  {
+    title: 'preferred version',
+    highlight: true,
+    render: (apiGroup: IAPIGroup) => {
+      return apiGroup.preferredVersion?.groupVersion;
+    },
+  },
+];
+
+export const ApiResources = () => {
+  const { entity } = useEntity();
+  const { value, error, loading } = useApiResources({
+    clusterName: entity.metadata.name,
+  });
+
+  return (
+    <>
+      {!value && loading && (
+        <>
+          <Skeleton />
+          <Skeleton />
+          <Skeleton />
+        </>
+      )}
+      {value && (
+        <Table
+          title="Api Resources"
```

Comment:
Since displaying yo the end user, maybe should be API Resources ? Since it is an acronym

Reply:
done